R/get_vaa.R

```diff
@@ -0,0 +1,82 @@
+# Primary HydroShare Data Resource
+vaa_hydroshare <-
+  'https://www.hydroshare.org/resource/6092c8a62fac45be97a09bfd0b0bf726/data/contents/nhdplusVAA.fst'
```

Comment:
Move this URL to nhdplusTools.R and add it too the package environ.

Reply:
moved the url